shotover-proxy/examples/redis-cluster-ports-rewrite/config.yaml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
Is there a reason we dont just use the standard config.yaml?

Reply:
Needs to be mounted for Shotover running in docker to use it.

Reply:
We discussed this and realized that it wont bitrot because the shotover container version is defined in the docker-compose.yaml file.
We can add a test for it in the future, but its not important.